app/javascript/apps/Project/index.tsx

```diff
@@ -0,0 +1,141 @@
+import {
+  Box,
+  Button,
+  CheckIcon,
+  EditIcon,
+  Flex,
+  Link,
+  LockIcon,
+  Sans,
+  Separator,
+  Tags,
+} from "@artsy/palette"
+import { Project, Stage, TagsList, Tags as TagsType } from "Typings"
+import { deployBlockPath, projectEditPath, tagPath } from "shared/UrlHelper"
+import { MainLayout } from "components/MainLayout"
+import React from "react"
+import { StageWithComparison } from "components/Stage/StageWithComparison"
+
+export const ProjectShow: React.FC<{ project: Project; tags: TagsType }> = ({
+  project,
+  tags,
+}) => {
+  const {
+    comparedStages,
+    description,
+    gitRemote,
+    block,
+    isKubernetes,
+    name,
+    severity,
+    orderedStages,
+  } = project
+  const isAgedClass = (severity > 10 && "aged") || ""
+  const blockLink = block && deployBlockPath(block.id)
+
+  return (
+    <MainLayout tags={tags}>
+      <Box
+        px={3}
+        py={3}
+        maxWidth={1080}
+        mx="auto"
+        position="relative"
+        className={isAgedClass}
+      >
+        <Box mb={1}>
+          <Sans size="10" style={{ textTransform: "capitalize" }}>
+            {name}
+          </Sans>
+          <Sans size="3">{description}</Sans>
+
+          <Box position="absolute" right={3} top={3}>
+            <Link href={projectEditPath(project.id)} underlineBehavior="hover">
+              <Flex alignItems="center">
+                <Sans size="3t" weight="medium" pr={0.5}>
+                  Edit
+                </Sans>
+                <EditIcon />
+              </Flex>
+            </Link>
+          </Box>
+
+          {gitRemote && (
+            <Box my={1}>
+              <Sans size="3">
+                <Link href={gitRemote}>{gitRemote}</Link>
```

Comment:
This looks like it will include the `.git` extension which isn't actually navigable. You could use a helper to transform it though.

Reply:
My mistake! It looks like Github redirects these paths successfully.